spyder/plugins/completion/languageserver/confpage.py

```diff
@@ -1034,6 +1784,82 @@ def setup_page(self):
         docstring_style_widget = QWidget()
         docstring_style_widget.setLayout(docstring_style_layout)
 
+        # --- Snippets tab ---
+        self.snippets_language = 'python'
+        grammar_url = (
+            "<a href=\"{0}/specifications/specification-current#snippet_syntax\">"
+            "{1}</a>".format(LSP_URL, _('the LSP grammar')))
+        snippets_info_label = QLabel(
+            _("Spyder allows to define custom completion snippets to use "
+              "in addition to the ones offered by the LSP. Each snippet "
```

Comment:
Let's change `LSP` here for `Language Server Protocol (LSP)`. That's because probably few of our users undestand what LSP means.